app/src/main/java/com/jalasoft/routesapp/ui/routes/viewModel/RoutesViewModel.kt

```diff
@@ -31,4 +45,144 @@ constructor(private val repository: RouteRepository) : ViewModel() {
         }
         return _routesList.value!!.size
     }
+
+    fun fetchDirections(startLocation: StartLocation, endLocation: EndLocation) = viewModelScope.launch {
+        val fetchedPlacesResponse = gDirectionsRepository.getDirections(startLocation, endLocation)
+        fetchedPlacesResponse.data?.let {
+            _directionsList.value = it
+            directionsList = _directionsList
+        }
+    }
+
+    fun getPossibleRoutes() = viewModelScope.launch {
+        val start1 = RouteAlgorithmFakeData.coordinatesToLocation(-16.52035351419114, -68.12580890707301)
```

Comment:
Add a // TODO:  Change this method when joining the locations with the routes here.

Reply:
Added TODO comment